index.js

```diff
@@ -255,7 +256,10 @@ client.on('message', async message => {
 			const role = message.member.roles.highest.name;
 			const name = message.member.displayName;
 			// let content = yourls.conditionalReplace(message.cleanContent, dependencies);
-			let content = message.cleanContent;
+			let content = emoji
+				.unemojify(content.cleanContent) // replace standard/unicode emojis
```

Comment:
Should be `message.cleanContent`, not `content.cleanContent`.

Reply:
oops